akka-http-core/src/main/scala/akka/http/impl/util/EnhancedConfig.scala

```diff
@@ -39,8 +39,8 @@ private[http] class EnhancedConfig(val underlying: Config) extends AnyVal {
     case x          ⇒ getIntBytes(path)
   }
 
-  def getPossiblyInfiniteLongBytes(path: String): Long = underlying.getString(path) match {
+  def getPossiblyInfiniteBytes(path: String): Long = underlying.getString(path) match {
     case "infinite" ⇒ Long.MaxValue
-    case x          ⇒ getIntBytes(path)
```

Comment:
actually had a bug in here, it would try to parse as Int, yet we need Long. It wasn't used in any place though :)